### PR TITLE
Update about fuzz search

### PR DESCRIPTION
**Pull request by piotr-symonowicz**

### CODE REVIEW

File: source/reference/stable/checkout-api/index.md [hunk 1]
```diff
@@ -429,6 +429,14 @@ The "filter" object has values for the field "swatch.desc" at the end of this JS
...
 In the filter object, the only thing that changes depending on what you filter on is the "count". If you do not filter on anything count = totalCount.
 
+### Searching improvements since v2.92
+
+Using the free-text `"search"` field in `POST /products` endpoint now has the following improvements:
+- Handling of non-Latin characters (e.g. "röd", "små"),  
+- Matching Latin to non-Latin characters (searching "röd" will also match "rod"),  
+- Proximity (fuzzy) matching of phrases (searching "prdct" should match "product").
+
+For example, if you have a Product named "Test Product" which contains "Röd" in its description, you should be able to find it using phrases like: `"test product"`, `"test röd"`, `"product rod"`, or even `"prdct rd"`.
```
Comment (arius86) [inline source/reference/stable/checkout-api/index.md:439]: `rd` won't work due to the limitation I set on the fuzzy match length, to make it faster I decided to do fuzzy search only for words longer than 3 letters :)

**Review by arius86 — CHANGES_REQUESTED** [review]

File: source/reference/stable/checkout-api/index.md (hunk 1 above)
Comment (piotr-symonowicz) [reply source/reference/stable/checkout-api/index.md:439]: Changing to `grön`, then :D

Comment (piotr-symonowicz) [reply source/reference/stable/checkout-api/index.md]: Wait... *Longer* than 3, or is 3 letters fine?

Comment (piotr-symonowicz) [reply source/reference/stable/checkout-api/index.md]: OK, updated with info on fuzzy only working above 3 chars

**Pull request merged**